README.md

```diff
@@ -144,7 +144,24 @@ _SoftiCAR GitHub Runner_ therefore employs [Sonatype Nexus](https://github.com/s
 
 ## 6 Architecture
 
-TODO
+The basic _SoftiCAR GitHub Runner_ architecture is depicted below:
+
+![](./docs/softicar-github-runner-architecture.png)
+
+Notes on components:
+
+- `runner` container:
+  - Uses the `sysbox` Docker runtime
+  - Is unprivileged
+  - Has restricted network access
+  - Has no bind-mounts with write access
+  - Is disposed and recreated after each build
```

Comment:
```suggestion
  - Is ephemeral, i.e. is disposed and recreated after each build
```

Reply:
This is problematic: In the context of GitHub Actions Runner, "ephemeral" only means that the runner de-registers itself after a single action (build). It does _not_ imply disposal and/or recreation. That's why the term was avoided here.